### ISSUE TITLE
Doesn't link out-of-the-box

### ISSUE THREAD

**Issue by SamB** [coapp-packages/SDL#2]
When I try to use this package without doing anything special, and include <SDL.h> before defining main(), MSVC 2013 gives me these errors:

1>------ Rebuild All started: Project: Win32Project1, Configuration: Debug Win32 ------
1>  main.c
1>MSVCRTD.lib(crtexew.obj) : error LNK2019: unresolved external symbol _WinMain@16 referenced in function ___tmainCRTStartup
1>C:\Users\Sam\Documents\Visual Studio 2013\Projects\Win32Project1\Debug\Win32Project1.exe : fatal error LNK1120: 1 unresolved externals
========== Rebuild All: 0 succeeded, 1 failed, 0 skipped ==========

**Comment by Raggles** [comment]
I don't think this is a problem with SDL, it looks like the linker can't find your entry point so I'd suggest you need to change some of your project settings.  Also I don't think the SDL package has VS2013 build targets

**Comment by ferzkopp** [comment]
Check the info here:
http://stackoverflow.com/questions/6626397/error-lnk2019-unresolved-external-symbol-winmain16-referenced-in-function

**Comment by SamB** [comment]
@Raggles Well, not SDL per se; upon further inspection, I suspect the problem is that this particular nuget package doesn't provide a "v120" build, only "v110" and "v100"... hmm... I suppose changing the Platform Toolset to "Visual Studio 2010 (v100)", it might actually work ... Yay, it works!

**Comment by madewokherd** [comment]
I've uploaded a new version with VS2013 and VS2015 builds.
